internal/workflow_testsuite.go

```diff
@@ -68,6 +68,11 @@ type (
 		runFn        func(args mock.Arguments)
 		waitDuration func() time.Duration
 	}
+
+	// TestContextPropagator propagates test keys across a workflow
```

Comment:
Why do we need this? Can we just use the public one?

Reply:
Done